Student_BCOs/Canyuan_Yang_BCO_000543.json

```diff
@@ -0,0 +1,242 @@
+{
+    "bco_id": "http://biocomputeobject.org/BCO_000543",
+    "checksum": "c3fc0f68fce596caf5ae18cbdb018760454012026c340c842ba116a304beeb6c",
+    "bco_spec_version": "1.3.0",
+    "provenance_domain": {
+        "name": "Use DEXSeq to study SJs and IRs",
+        "version": "2.0.0",
+        "license": "https://creativecommons.org/licenses/by/4.0/",
+        "created": "2019-10-09T01:55:40.046696",
+        "modified": "2019-10-17T18:43:11.311551",
+        "review": [
+            {
+                "status": "in-review",
+                "reviewer_comment": "Reviewed by BioCompute technical assistant, approval pending modification by author",
+                "reviewer": {
+                    "orcid": "https://orcid.org/0000-0002-8824-4637",
+                    "affiliation": " George Washington University",
+                    "contribution": [
+                        "curatedBy"
+                    ],
+                    "name": "Janisha Patel",
+                    "email": "janishapatel@gwu.edu"
+                },
+                "date": "2019-10-17"
+            }
+        ],
+        "contributors": [
+            {
+                "name": "Canyuan Yang",
+                "orcid": "",
+                "affiliation": " George Washington University",
+                "contribution": [
+                    "createdBy"
+                ],
+                "email": "canyuanyang@gwu.edu"
+            },
+            {
+                "name": "Yafang Li",
+                "orcid": "",
+                "affiliation": "Dartmouth College",
+                "contribution": [
+                    "authoredBy"
+                ],
+                "email": ""
+            },
+            {
+                "name": "Xiayu Rao",
+                "orcid": "",
+                "affiliation": "The University of Texas MD Anderson Cancer Center",
+                "contribution": [
+                    "authoredBy"
+                ],
+                "email": ""
+            },
+            {
+                "name": "William W. Mattox",
+                "orcid": "",
+                "affiliation": "The University of Texas MD Anderson Cancer Center",
+                "contribution": [
+                    "authoredBy"
```

Comment:
All publication authors should both "derivedFrom" and "authoredBy" contribution types. Please add.. See #113